sagemaker-pipeline-parameterization/notebook.ipynb

```diff
@@ -0,0 +1,820 @@
+{
```

Comment:
<p>Why is "Standard deviation" NaN for all values in both images? If this is intentional, add a note explaining why.</p><p> </p><p>Also recommended to add a few sentences describing the different components and how to interpret them. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3021/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>Because there is no standard deviation here - it's mandatory to include in the metrics object but there is no standard deviation.</p>

Reply:
Worth adding a brief note like "The standard deviation shows as NaN since it is not relevant to this model's calculated metrics"? Could seem like something is incomplete or incorrect if a customer does not examine in detail. But fine to leave as-is if you prefer.